src/Person.jsx

```diff
@@ -0,0 +1,19 @@
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">My name is {name}</h2>
+      {age && <p className="Person__age">I am {age}</p>}
+      {isMarried ? (
+        <p className="Person__partner">
+          {sex === 'm'
+            ? `${partnerName} is my wife`
+            : `${partnerName} is my husband`}
+        </p>
+      ) : (
+        <p className="Person__partner">I am not married</p>
+      )}
```

Comment:
you can simplify this and render just one `p`  and show the data according to the condition

```suggestion
         <p className="Person__partner">
            {isMarried
              ? -----------------
              : 'I am not married'}
         </p>
```